### PR TITLE
Update attributes to reflect API changes

### PR DESCRIPTION
**Pull request by yuderekyu**
Takes [these updates](https://goshippo.com/docs/changelog) into account, updating attributes to reflect  API changes. Fixes https://github.com/coldbrewcloud/go-shippo/issues/8.

### CODE REVIEW

File: _examples/address_validation/main.go [hunk 1]
```diff
@@ -5,8 +5,8 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/coldbrewcloud/go-shippo"
-	"github.com/coldbrewcloud/go-shippo/models"
+	"github.com/yuderekyu/go-shippo"
```
Comment (d5) [inline _examples/address_validation/main.go]: Can you please fix import paths?

**Comment by d5** [comment]
Thanks for the PR. This is a big change. I will take a look.

**Comment by d5** [comment]
Looks like they had two additional break changes since I started this project. The current code is based on version `2014-02-11`. Probably I will make a branch for that if we merge this PR.

**Comment by d5** [comment]
@yuderekyu you will have to revert all import path changes from `github.com/yuderekyu/go-shippo` back to `github.com/coldbrewcloud/go-shippo`. Otherwise I cannot accept this PR.

**Comment by yuderekyu** [comment]
@d5 I wasn't sure if you were still maintaining this, so I was planning on just building off of my fork and close this later. But yeah, I'll go ahead and revert those back in order to merge this in.

**Comment by yuderekyu** [comment]
@d5 do you have an update on this PR? I have a project that's dependent on this wrapper, so I will revert the imports back to my fork if you decide to not merge this in.

**Comment by d5** [comment]
Well there are still some imports that's not `coldbrewcloud/go-shippo`. Can you revert them too?

**Comment by yuderekyu** [comment]
@d5 Apologies, I missed those. Should be good now!

**Pull request merged**